### DISCUSSION TITLE
Broken ARIA reference accessibility error in wave tool

### DISCUSSION THREAD

**Discussion by sachinsahu731** [d3/d3#3565] [Q&A]
We are trying to make the data is accessible by screen reader and keyboard and that is working fine after using this 
https://www.a11ywithlindsey.com/blog/accessibility-d3-donut-charts

But in wave tool (checking all accessibility validation) 

`.attr('aria-describedby', (d, i) => `tooltip-${i}`)`

is throw an error **Broken ARIA reference**
Because the aria-described is needed html element with same id and that is not exist in dom .
Kindly help on how to fix this issue . PFA for more reference.
![Screenshot (37)](https://user-images.githubusercontent.com/29474792/162926209-fa8a8827-0b9a-4ab2-aae8-560a219f5bc4.png)

![screenshot (7)](https://user-images.githubusercontent.com/29474792/162926387-e96334fa-bf99-405c-8262-c70a8b576f0e.png)

**Comment by curran** [comment]
Is this an issue with D3 or an issue with "wave tool"? How to reproduce the issue?

**Reply by sachinsahu731** [reply]
@curran  The issue with in d3 only .Using aria-describedby have to same id in element .
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby

**Reply by mbostock** [reply]
This is not an issue with D3.
